xchange-core/src/main/java/org/knowm/xchange/dto/marketdata/OrderBook.java

```diff
@@ -25,6 +26,8 @@
   private final List<LimitOrder> bids;
   /** the timestamp of the orderbook according to the exchange's server, null if not provided */
   private Date timeStamp;
+  /** the metadata, possibly provided by the api */
+  private Map<String, Serializable> metadata;
```

Comment:
this data structure is to loose

if you need exchange specific fields then use methods from *Raw classes

Reply:
reverted

Reply:
reverted